src/System.Private.ServiceModel/src/project.json

```diff
@@ -50,12 +50,14 @@
   },
```

Comment:
Thanks @ericstj. Should we update versions of packages below in src/project.json here too to be consistent with the change you made in src/windows/project.json?

"System.Collections.Concurrent": "4.0.10",
"System.ObjectModel": "4.0.10",
"System.Xml.XmlSerializer": "4.0.10"